### PR TITLE
fix docker compose

### PR DESCRIPTION
**Pull request by sevenzing**

### CODE REVIEW

**Review by rimrakhimov — APPROVED** [review]

File: verification/docker-compose.yaml [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+    ## you can build image from source if you want
+    # build: .
+    image: ghcr.io/blockscout/verification:latest
     ports:
       - "8043:8043"
     command: -c /app/config.toml
     volumes:
-      - ./example_config.toml:/app/config.toml
\ No newline at end of file
+      - ./example_config.toml:/app/config.toml
```
Comment (leviska) [inline verification/docker-compose.yaml:11]: add
```yaml
# optional: you can provide custom config
 - ./example_config.toml:/app/config.toml
# optional: provide volume or folder to store compilers between launches
 - ./compilers:/app/compilers
```

File: verification/docker-compose.yaml [hunk 2]
```diff
@@ -1,9 +1,12 @@
 version: "3"
 services:
   verification:
-    build: .
+    ## you can build image from source if you want
```
Comment (leviska) [inline verification/docker-compose.yaml:4]: `# you can build image locally, or use prebuilt images from registry`

File: verification/docker-compose.yaml (hunk 1 above)
Comment (leviska) [reply verification/docker-compose.yaml:11]: ah, not providing the config file will not work because of the `-c /app/config.toml`, isn't it? :(

Comment (leviska) [reply verification/docker-compose.yaml:11]: then may be add before command
``# you can run the service with default config, for that you need to remove `command` parameter``

**Review by leviska — APPROVED** [review]

**Pull request merged**